requirements/requirements.in

```diff
@@ -68,4 +68,4 @@ mpld3
 
 # External Resource Plugins
 ## Ray
-ray>=2.2.0
\ No newline at end of file
+ray[default]>=2.2.0
```

Comment:
the ray dashboard that gets created for each cluster wasn't working without this

Reply:
Why is that? Do we need to mention this in our Ray integration docs?

Reply:
Can we expose this dashboard to users during the duration of the cluster's lifetime?

Reply:
> Why is that? Do we need to mention this in our Ray integration docs?

Because the dashboard requires some deps you don't get if you just do `ray` without the `[default]`. And no we don't need to mention it in our docs because I also changed `sematic/ee/extras.bzl` to say that `sematic[ray]` depends on `ray[default]`